tests/common.go

```diff
@@ -5131,3 +5131,68 @@ func GetPoolIDsFromVolName(volName string) ([]string, error) {
 	}
 	return poolUuids, err
 }
+
+// WaitTillEnterMaintenance Wait till Node Enters maintenance mode
+func WaitTillEnterMaintenanceMode(n node.Node) {
+	t := func() (interface{}, bool, error) {
+		if Inst().V.IsNodeInMaintenance(n) == true {
+			return nil, true, nil
+		}
+		return nil, false, fmt.Errorf("Not in Maintenance mode")
+	}
+
+	_, err := task.DoRetryWithTimeout(t, 20*time.Minute, 2*time.Minute)
+	log.FailOnError(err, fmt.Sprintf("Failed to Enter maintenence mode on node [%s]", n.Name))
```

Comment:
Can we have this function return and then handle `log.FailOnError()` where you call it

Reply:
handling error from script